tests/src/cosmwasm.spec.ts

```diff
@@ -327,6 +376,67 @@ test.serial("Happy Path (cross-stake / cross-unstake)", async (t) => {
   });
   console.log("List of staked tokens on consumer chain: ", emptyStakedTokenResponse);
 
+  /** Start withdraw rewards process */
```

Comment:
surprised there are no longer wait blocks to ensure we get more staking rewards piled up.

I don't have the headspace to review it properly now, but think we could extend these cases to make it more realistic. I would defer to @JakeHartnell for now, and consider this food for thought in a future PR. This is good enough and shows nothing breaks.